src/components/Common/UI/Breadcrumb/Breadcrumb.tsx

```diff
@@ -0,0 +1,16 @@
+import { Breadcrumb as AriaBreadcrumb } from 'react-aria-components'
+import type { BreadcrumbProps } from './BreadcrumbTypes'
+
+export function Breadcrumb({ children, className, isCurrent, href, onClick }: BreadcrumbProps) {
+  return (
+    <AriaBreadcrumb className={className} data-current={isCurrent}>
+      {href ? (
+        <a href={href} onClick={onClick}>
+          {children}
+        </a>
+      ) : (
+        <span>{children}</span>
+      )}
```

Comment:
So this kinda brings us back to our discussion the other day... should this component actually be referencing the `Link` component from component context here instead? I would say at a minimum this should be using that component directly so that we can keep our link styles centralized